onadata/libs/tests/utils/test_viewer_tools.py

```diff
@@ -164,3 +165,13 @@ def test_get_form_url(self):
         # with form pk url http://ona.io/bob/1
         url = get_form_url(request, username='bob', xform_pk=1)
         self.assertEqual(url, 'https://ona.io/bob/1')
+
+    @override_settings(TESTING_MODE=False)
+    def test_get_submissions_url(self):
+        """Test get_submissions_url()."""
+        request = RequestFactory().get('/')
+        url = get_submission_url(
+            request, username="milly", id_string="tag_team")
+        __import__('ipdb').set_trace()
+        self.assertEqual(
+            url, None)
```

Comment:
Will be nicer to test the other segments of the function....like the errors section